### PR TITLE
fix: minor fixes 

### PR DESCRIPTION
**Pull request by peterambrus3**
fix(agent): inject secret key
fix(crux-ui): confirmation modal cancel fix
fix(crux-ui): deployment delete navigate
fix(crux-ui): image config sections improvements
fix(crux-ui): image list actions responsive display
feat(crux-ui): add back btn to image details
fix(crux-ui): node status indicator position center
fix(crux): add common error interceptor

### CODE REVIEW

File: golang/pkg/dagent/config/secret.go [hunk 1]
```diff
@@ -4,11 +4,13 @@ import (
 	"fmt"
 
 	config "github.com/dyrector-io/dyrectorio/golang/internal/config"
+	"github.com/rs/zerolog/log"
 )
 
 type ConfigFromFile string
 
 func (field *ConfigFromFile) SetValue(secretPath string) error {
+	log.Debug().Msgf("loading config from file: %v", secretPath)
```
Comment (polaroi8d) [inline golang/pkg/dagent/config/secret.go:13]: We are using `Str(key, value)` and `Msg()` for printing out variables. (I just want to point out)

**Comment by polaroi8d** [comment]
Some linting errors, but okay for me.

**Review by balanceee — APPROVED** [review]
LGTM. Tested locally.

File: golang/pkg/dagent/config/secret.go (hunk 1 above)
Comment (peterambrus3) [reply golang/pkg/dagent/config/secret.go]: Done, thx.

File: web/crux/src/services/domain.notification.service.ts [hunk 2]
```diff
@@ -47,7 +44,17 @@ export default class DomainNotificationService {
         },
       })
 
-      await Promise.all(notifications.map(it => this.send(it.url, it.type, template)))
+      if (notifications.length > 0) {
+        try {
+          const identity = await this.kratos.getIdentityById(template.identityId)
+          template.message.owner = nameOrEmailOfIdentity(identity)
+        } catch {
+          this.logger.error(`Identity: "${template.identityId}" not found, cant set template message owner!`)
```
Comment (robot9706) [inline web/crux/src/services/domain.notification.service.ts:52]: Minor grammar: `cant` -> `can't`

File: web/crux-ui/src/components/products/versions/images/config/crane-config-section.tsx [hunk 3]
```diff
@@ -35,9 +35,15 @@ const CraneConfigSection = (props: CraneConfigSectionProps) => {
     propsOnChange(newConfig)
   }
 
-  const contains = (filter: ImageConfigFilterType): boolean => filters.indexOf(filter) !== -1
+  const contains = (filter: CraneConfigFilterType, checkAny: boolean = false): boolean => {
+    if (checkAny) {
```
Comment (balanceee) [inline web/crux-ui/src/components/products/versions/images/config/crane-config-section.tsx:39]: Personally i dont like mixing these kind of checks. Please extract it to another function. I think you make this function to be as common as possible and move it to the images.ts.

File: web/crux-ui/src/components/products/versions/images/config/dagent-config-section.tsx [hunk 4]
```diff
@@ -37,9 +37,15 @@ const DagentConfigSection = (props: DagentConfigSectionProps) => {
     propsOnChange(newConfig)
   }
 
-  const contains = (filter: ImageConfigFilterType): boolean => filters.indexOf(filter) !== -1
+  const contains = (filter: DagentConfigFilterType, checkAny: boolean = false): boolean => {
```
Comment (balanceee) [inline web/crux-ui/src/components/products/versions/images/config/dagent-config-section.tsx:40]: Same as above

File: web/crux-ui/src/components/products/versions/images/config/common-config-section.tsx [hunk 5]
```diff
@@ -43,7 +43,13 @@ const CommonConfigSection = (props: CommonConfigSectionProps) => {
 
   const [config, setConfig] = useState<CommonConfigDetails>(propsConfig)
 
-  const contains = (filter: ImageConfigFilterType): boolean => filters.indexOf(filter) !== -1
+  const contains = (filter: CommonConfigFilterType, checkAny: boolean = false): boolean => {
```
Comment (balanceee) [inline web/crux-ui/src/components/products/versions/images/config/common-config-section.tsx:46]: Same as under

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: web/crux/src/services/domain.notification.service.ts (hunk 2 above)
Comment (peterambrus3) [reply web/crux/src/services/domain.notification.service.ts:52]: fixd

**Review by robot9706 — APPROVED** [review]
LGTM🚀

**Review by balanceee — APPROVED** [review]
LGTM

**Review by nandor-magyar — APPROVED** [review]

File: web/crux-ui/src/components/products/versions/images/config/crane-config-section.tsx (hunk 3 above)
Comment (peterambrus3) [reply web/crux-ui/src/components/products/versions/images/config/crane-config-section.tsx]: refactored thx

**Pull request merged**